src/main/java/se/sundsvall/datawarehousereader/api/model/installation/InstallationDetails.java

```diff
@@ -0,0 +1,262 @@
+package se.sundsvall.datawarehousereader.api.model.installation;
+
+import static io.swagger.v3.oas.annotations.media.Schema.AccessMode.READ_ONLY;
+
+import java.time.LocalDate;
+import java.util.List;
+import java.util.Objects;
+
+import se.sundsvall.datawarehousereader.integration.stadsbacken.model.installation.InstallationMetaDataEmbeddable;
+
+import io.swagger.v3.oas.annotations.media.ArraySchema;
+import io.swagger.v3.oas.annotations.media.Schema;
+
+@Schema(description = "Installation details model")
+public class InstallationDetails {
+
+	@Schema(description = "Company", example = "Sundsvall Energi AB", accessMode = READ_ONLY)
+	private String company;
+
+	@Schema(description = "Type", example = "Fjärrvärme", accessMode = READ_ONLY)
+	private String type;
+
+	@Schema(description = "Facility id", example = "735999109270751042", accessMode = READ_ONLY)
+	private String facilityId;
+
+	@Schema(description = "Placement id", example = "5263", accessMode = READ_ONLY)
+	private int placementId;
+
+	@Schema(description = "Care of address", example = "Agatha Malm", accessMode = READ_ONLY)
+	private String careOf;
+
+	@Schema(description = "Street", example = "Storgatan 9", accessMode = READ_ONLY)
+	private String street;
+
+	@Schema(description = "Postal code", example = "85230", accessMode = READ_ONLY)
+	private String postCode;
```

Comment:
Should the attribute be named `postalCode` (as in the description)?

Reply:
Postcode is the name in the database view, and therefore it is just copied over.

Reply:
In the vInvoice (also in this PR) it's called `postalCode`.
In the CustomerDetails API-model, it's called `postalCode`

Regardless of what the attribute is called in the DB, we should be consistent in the API, across all resources, i.m.h.o.

If you choose `postCode` everywhere, that's fine, but then I would suggest to change the description as well (different words for the same thing just introduce confusion).